README.md

```diff
@@ -97,11 +99,24 @@ curl -s -o /dev/null -w "%{http_code}\n" -X POST -H "Authorization: Bearer ${API
 
 6. Go to the [System Tests Workflow](https://github.com/MAAP-Project/maap-data-system-tests/actions/workflows/test.yml) page to see that the workflow is running.
 
-## Running the notebook manually
+## Running the Notebook Manually
 
 1. Install the Python dependencies with `pip install -r requirements.txt`
```

Comment:
Do these dependencies also include papermill (and its dependants)? If not, do we need to add more here

Reply:
papermill is already installed in the base docker image

Reply:
We should probably add the "docker run" command in this as another step to be absolutely clear what needs to be run, as at first glace, I wasn't aware docker was even involved (putting my outside person hat on).

Reply:
I don't think these manual instructions even still work. We should put those changes in a separate PR, since this one was just a stylistic change to the name of the section.

Reply:
Fair counter. Approved.

Reply:
sorry, I forgot there are 2 separate requirements.txt -- the one in the root is the one that these instructions refer to and does install papermill. The other one (that was modified in the PR) is the requirements.txt for running the python script that invokes GH Action to run the tests, which doesn't need it.